### DISCUSSION TITLE
Brain achievement

### DISCUSSION THREAD

**Discussion by BitTheCat** [Schweinepriester/github-profile-achievements#145] [Q&A] [answered]
@vzsoares for u

**Comment by vzsoares** [comment]
super duper complex answer from aliment brusque beneathfurthermore thereupon may abroad in spite of authorize worthlessness to summarise peevish verbose say my thou thirdly in like manner inspite of .

**Comment by thenithinbalaji** [comment, marked as answer]
The brain is a complex organ that controls thought, memory, emotion, touch, motor skills, vision, breathing, temperature, hunger and every process that regulates our body. Together, the brain and spinal cord that extends from it make up the central nervous system, or CNS.

**Reply by BitTheCat** [reply]
i see u 2 have the bronze level.
I didn't receive the bronze yet xD

**Reply by BitTheCat** [reply]
I mark your answer too ^^

**Reply by thenithinbalaji** [reply]
You have answered 5 discussions, but have not yet got galaxy brain

**Reply by thenithinbalaji** [reply]
That is because you might have marked your own answer as the answer to some question. Remove them. @BitTheCat

**Reply by BitTheCat** [reply]
Yes but I don't know why

**Reply by BitTheCat** [reply]
Oh ok i try ty

**Reply by BitTheCat** [reply]
I remove it, now i need to receive other 2? @thenithinbalaji

**Reply by thenithinbalaji** [reply]
Are you sure you removed all your answers which you marked as answer

**Reply by BitTheCat** [reply]
yes

**Reply by thenithinbalaji** [reply]
then I am not sure why, try answering 2 questions

**Reply by BitTheCat** [reply]
@thenithinbalaji would you create 2 and let's try?

**Reply by thenithinbalaji** [reply]
sure
